style.css

```diff
@@ -93,10 +79,36 @@ img:hover{
   float: left;
 }
 
-#closing{
-  color:rgb(231,84,128);
+#closing {
+  color: rgb(231, 84, 128);
 }
 
-#candidates{
+#candidates {
   border: 10px solid pink;
-}
\ No newline at end of file
+}
+
+@media screen and (max-width: 600px) {
+  #tribute-link {
+    position: initial;
+    left: initial;
+    top: initial;
+  }
+  .column {
+    width: 100%;
+    float: none;
+    padding: 30px;
+    height: 300px;
+    margin-top: 50px;
+  }
+  .img-row {
+    display: block;

```

Comment:
Changed this back to block as on desktop, you set `display:flex` to enable the split column layout, which isn't needed for mobile